api/app/controllers/spree/api/v2/storefront/stores_controller.rb

```diff
@@ -3,12 +3,26 @@ module Api
     module V2
       module Storefront
         class StoresController < ::Spree::Api::V2::ResourceController
+          before_action :update_options
+
           def current
             render_serialized_payload { serialize_resource(current_store) }
           end
 
+          def export_google_rss_feed
+            send_data export_google_rss, filename: 'products.rss', type: 'text/xml'
+          end
+
           private
 
+          def update_options
+            @settings = Spree::GoogleFeedSetting.find_by(spree_store_id: current_store)
+          end
+
+          def export_google_rss
```

Comment:
```suggestion
          def export_google_rss_service
```
To be clear that this method is a wrapper for the service.